repos/system_upgrade/el7toel8/actors/persistentnetnamesconfig/tests/test_persistentnetnamesconfig.py

```diff
@@ -1,7 +1,30 @@
+import json
+import os
+
+import pytest
+
 from leapp.libraries.actor import persistentnetnamesconfig
-from leapp.models import PersistentNetNamesFacts, PersistentNetNamesFactsInitramfs
-from leapp.models import RenamedInterface, RenamedInterfaces, InitrdIncludes
-from leapp.models import Interface, PCIAddress
+from leapp.libraries.common.testutils import CurrentActorMocked, logger_mocked, produce_mocked
+from leapp.models import (
+    InitrdIncludes,
+    Interface,
+    PCIAddress,
+    PersistentNetNamesFacts,
+    PersistentNetNamesFactsInitramfs,
+    RenamedInterface,
+    RenamedInterfaces,
+)
+
+TEST_DIR = os.path.dirname(os.path.abspath(__file__))
+CUR_DIR = ""
+
+
+@pytest.fixture
```

Comment:
Hmm sorry if I got it wrong, are you using adjust_cwd mainly for figuring out full path to proper crashed_interfaces.json?
In that case you could theoretically use PYTEST_CURRENT_TEST https://stackoverflow.com/questions/55913423/pytest-get-the-file-path-of-current-test-file

Reply:
I am using that as it was told earlier we should use it. Maybe I missed something? (@ZhukovGreen)
Anyway, as far as it works now, let's keep it.